components/basketball/homePage/QuickLinks.tsx

```diff
@@ -0,0 +1,44 @@
+import Link from "next/link";
+import React from "react";
+
+interface QuickLinksProps {}
+const competitions = ["men-open", "women-open", "men-senior"];
```

Comment:
would be better to rely on the `getReasonSeasons` to get the competition. In case when the competition type/name changed